setup.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+version = "0.2.0"
+
+def _read_long_desc():
+    with open("README.md") as fp:
+        return fp.read()
+
+setup(
+  name="tern",
+  version=version,
+  author="VMWare Inc",
+  author_email="FIXME@FIXWHAT.THEEMAIL",
```

Comment:
I can cherry-pick and replace these values if needed

Reply:
Yeah, I wasn't sure what to put in here...